repos/fdbt-site/src/pages/products/services.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { ReactElement } from 'react';
+import { MyFaresService, NextPageContextWithSession } from '../../interfaces/index';
+import { BaseLayout } from '../../layout/Layout';
+import { getBodsServicesByNoc } from '../../data/auroradb';
+import { getAndValidateNoc } from '../../utils';
+import { myFaresEnabled } from '../../constants/featureFlag';
+import moment from 'moment';
+
+const title = 'Services - Create Fares Data Service';
+const description = 'View and access your services in one place.';
+
+interface ServicesProps {
+    services: MyFaresService[];
+    myFaresEnabled: boolean;
+}
+
+const Services = ({ services, myFaresEnabled }: ServicesProps): ReactElement => {
+    return (
+        <>
+            <BaseLayout
+                title={title}
+                description={description}
+                showNavigation
+                referer={''}
+                activePage="services"
+                myFaresEnabled={myFaresEnabled}
+            >
+                <div className="govuk-grid-row">
+                    <div className="govuk-grid-column-full">
+                        <h1 className="govuk-heading-xl govuk-!-margin-bottom-3">Services</h1>
+
+                        {ServicesTable(services)}
+                    </div>
+                </div>
+            </BaseLayout>
+        </>
+    );
+};
+
+const ServicesTable = (services: MyFaresService[]): ReactElement => {
```

Comment:
As this is just a function in should be in camel case. If it were a full on react component with props then it could be like this but would have to be explicitly tested elsewhere